### PR TITLE
18: add logger message for connection established time

### LINKED ISSUE #18: Add time measuring for create connection statements

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
Closes #18

### CODE REVIEW

File: src/main/java/com/exasol/adapter/jdbc/JdbcAdapter.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ private SqlDialect createDialect(final Connection connection, final AdapterPrope
 
     @Override
     public DropVirtualSchemaResponse dropVirtualSchema(final ExaMetadata metadata,
-            final DropVirtualSchemaRequest request) {
+                                                       final DropVirtualSchemaRequest request) {
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/jdbc/JdbcAdapter.java]: This indentation does not look right. What happened here?

File: src/main/java/com/exasol/adapter/jdbc/RemoteConnectionFactory.java [hunk 2]
```diff
@@ -71,15 +72,16 @@ private Connection createConnection(final String connectionName, final ExaMetada
     }
 
     private Connection establishConnectionWithKerberos(final String password, final String username,
-            final String address) throws SQLException {
+                                                       final String address) throws SQLException {
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/jdbc/RemoteConnectionFactory.java]: Same here.

File: src/main/java/com/exasol/adapter/jdbc/JdbcAdapter.java (hunk 1 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/jdbc/JdbcAdapter.java]: done

File: src/main/java/com/exasol/adapter/jdbc/RemoteConnectionFactory.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/jdbc/RemoteConnectionFactory.java]: done

**Pull request merged**